draft-js-anchor-plugin/src/components/LinkButton/AddLinkForm.js

```diff
@@ -68,6 +69,10 @@ export default class AddLinkForm extends Component {
     this.onClose();
   }
 
+  isUrl(value) {
+    return this.props.validateUrl && this.props.validateUrl(value) || !this.props.validateUrl && URLUtils.isUrl(value);
```

Comment:
How would you do that? my idea was to use the custom validation function if there is one, otherwise use the default validation (the one in URLUtils) that was used before this PR.

Reply:
isURL(value) {
   if (props.validateUrl) { return props.validateUrl(value) }

   return URLUtils.isUrl(value);
}

Reply:
That looks better! thanks!